packages/bpk-component-banner-alert/package.json

```diff
@@ -20,6 +20,6 @@
     "bpk-react-utils": "^2.6.1",
     "bpk-tokens": "^27.0.4",
     "prop-types": "^15.5.8",
-    "react-transition-group": "^1.2.1"
+    "react-transition-group": "^2.4.0"
```

Comment:
This needs to be upgraded... see https://github.com/reactjs/react-transition-group/blob/master/CHANGELOG.md#v200